applications/eigen_solvers_application/custom_python/add_custom_solvers_to_python.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+//  KRATOS _______
+//        / ____(_)___ ____  ____
+//       / __/ / / __ `/ _ \/ __ \
+//      / /___/ / /_/ /  __/ / / /
+//     /_____/_/\__, /\___/_/ /_/ SolversApplication
+//             /____/
+//
+//  Author: Thomas Oberbichler
+*/
+
+// System includes
+
+// External includes
+#include <boost/python.hpp>
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "custom_python/add_custom_solvers_to_python.h"
+
+#include "spaces/ublas_space.h"
+#include "linear_solvers/linear_solver.h"
+#include "custom_solvers/EigenDirectSolver.h"
```

Comment:
eigen_direct_solver.h would be style compatible

Reply:
Also the class name?

Reply:
The class name should be CamelCase: `EigenDirectSolver`